webhook.go

```diff
@@ -300,6 +303,30 @@ func (v1 *RegistrationV1) ValidateDuration(ttl time.Duration) error {
 	return errs
 }
 
+func (v1 *RegistrationV1) CheckUntil(now func() time.Time, jitter, maxTTL time.Duration) error {
```

Comment:
all of this code comes from https://github.com/xmidt-org/ancla/blob/e805e4b4ede085fb97b4fcab8ca7a1ebc2216520/webhookValidator.go#L113